### ISSUE TITLE
Keyframes prefixes

### ISSUE THREAD

**Issue by klimashkin** [JPeer264/node-rename-css-selectors#8]
I tried to process minified `bootstrap.min.css` (could be a good source to add to test, btw) with `node-rename-css-selectors`, and noticed that prefixed `@keyframes` names are not transformed.
```css
@-webkit-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}
@-o-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}
@keyframes t{from{background-position:40px 0}to{background-position:0 0}}
hl{
  -webkit-animation:t 2s linear infinite;
  -o-animation:t 2s linear infinite;
  animation:t 2s linear infinite
}
```

**Comment by JPeer264** [comment]
That's indeed a good hint, I will take a look at `bootstrap.min.css`

**Comment by JPeer264** [comment]
Fixed in v`1.2.3`
